src/state/update-user.js

```diff
@@ -0,0 +1,23 @@
+// import { useEffect, useState } from "react";
+
+export const updateUser = (uname, user) => {
+  console.log("there");
+  console.log("uname", uname, "user", user);
+  fetch(`https://m07.herokuapp.com/${uname}`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    mode: "cors",
+    body: JSON.stringify(user)
+  })
+    .then(response => response.json())
+    .then(() => {
+      console.log("now");
```

Comment:
This should be exposed as react hook

Reply:
@marcopeg Should it be useEffect() instead?

Reply:
It doesn't have to useEffect as a MUST, it hust has to follow the react hooks structure and guidelines. Follow the documentation.